tools/crm_simulate.c

```diff
@@ -365,13 +358,16 @@ static GOptionEntry source_entries[] = {
 static void
 get_date(pe_working_set_t *data_set, bool print_original, char *use_date)
 {
+    pcmk__output_t *out = data_set->priv;
     time_t original_date = 0;
 
     crm_element_value_epoch(data_set->input, "execution-date", &original_date);
 
     if (use_date) {
         data_set->now = crm_time_new(use_date);
-        quiet_log(" + Setting effective cluster time: %s", use_date);
+        if (!out->is_quiet(out)) {
+            out->info(out, "Setting effective cluster time: %s", use_date);
+        }
```

Comment:
This would be nice, but there's a couple places on master that just have to be different.  For example:
```
    if (!out->is_quiet(out)) {
        out->info(out, "Status of %s: '%s' %s %s", crm_str(sys_from),
                  crm_str(state), (!pcmk__str_empty(last_updated))?
                  "last updated":"", crm_str(last_updated));
    } else {
        out->info(out, "%s", crm_str(state));
    }
```

If we can decide on a strategy for the three or four locations like that (`grep -r is_quiet -A 1 *`), I'm all for this.

Reply:
I'm thinking custom messages.

BTW ack will make your life a lot easier than grep :)

Reply:
There are a lot of places where out->info is called, most prominently in crm_resource and crm_mon.  Previously, I had been using out->info for stuff like various status messages, explaining that we're reconnecting to the daemons, etc.  Doing this will cause all those messages to go away, though if the user adds -Q that's probably what they want.  It might also be worth revisiting the purpose of all those uses of out->info.  Maybe some of them should turn into warnings, or just go away altogether.

Reply:
> Doing this will cause all those messages to go away, though if the user adds -Q that's probably what they want.

That's what gave me the idea. As a general rule, the existing implementation of --quiet across various tools dispenses with all output except some essential result. Having info() automatically respect it could simplify a lot of that and make the tools behave consistently. Not all tools currently support --quiet, but I think we're moving in that direction.

Reply:
It does mean we need custom messages for any usage that just produces less verbose output for quiet, but maybe that's worth it